src/js/heuristicblocking.js

```diff
@@ -457,6 +607,23 @@ function _extractCookies(details) {
   return cookies;
 }
 
+
+function _extractArgs(details) {
+  if (details.method == "POST" && details.requestBody) {
+    return details.requestBody.formData;
+  }
+
+  let args = {};
+  if (details.method == "GET") {
+    let url = new URL(details.url);
```

Comment:
`URI()` (some custom thing from legacy ABP we use) vs. `URL()` (browser built-in)? We eventually want to remove `URI` probably, but not as part of this PR. I guess we use `new URL` elsewhere already, so not a new inconsistency. Does one over the other matter in this particular case?